### PR TITLE
upgrade to the latest node verison

### PR DESCRIPTION
**Pull request by katelynsills**
## Description
Upgrades to the latest node version, which is 9.0.0 at the time of this commit

## Test Plan
yarn start
yarn webpack

### CODE REVIEW

**Comment by karenmrose** [comment]
Awesome! Do you mind updating the version in the travis.yml file as well?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,31 +20,28 @@ brew install git
...
+nvm install 9.0.0
 ````
 
 ## Install Node Modules and Global Dependencies
 ````
 yarn
-yarn global add jest@v16.0.2
 yarn global add nodemon
-yarn global add webpack@v2.1.0-beta.25
-yarn global add body-parser
```
Comment (karenmrose) [inline CONTRIBUTING.md]: Just out of curiosity, could you maybe elaborate on the reasoning for these documentation updates (i.e. adding/removing certain commands)? Thanks!

Comment (katelynsills) [reply CONTRIBUTING.md]: Sure, we could add in webpack and body-parser as global installs. That might have been a mistake on my part to take them out. My thought was that we made the documentation when we were still using npm for everything, so I was trying to change the docs to 1) follow the recommended install guides for yarn and nvm and 2) match the linux part of the docs.

Comment (karenmrose) [reply CONTRIBUTING.md]: Oh good point haha. Yeah we don't want those 3 installed globally.

**Comment by katelynsills** [comment]
Good catch! Added in the travis.yml change

**Pull request merged**